.github/workflows/test-convene-web.yml

```diff
@@ -13,8 +13,12 @@ jobs:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
       PORT: 80
-      APP_URL: http://localhost
+      APP_ROOT_URL: http://localhost
```

Comment:
Because we never have a `APP_URL` env var, the correct env var is `APP_ROOT_URL`.
https://github.com/zinc-collective/convene/blob/fba8f8295f19e9b1d2a75c6efe05f88001034861/convene-web/.env.example#L24
https://github.com/zinc-collective/convene/blob/fba8f8295f19e9b1d2a75c6efe05f88001034861/.env.example#L1-L4

We should be ok since `APP_URL` was only in CI.

Reply:
@anaulin error makes me rethink this, but then I don't have a `.env` at root, I only have `convene-web/.env` which doesn't have `APP_URL`, only `APP_ROOT_URL`.